dropwizard-app/src/main/java/org/coner/boundary/CompetitionGroupBoundary.java

```diff
@@ -0,0 +1,34 @@
+package org.coner.boundary;
+
+import org.coner.core.domain.CompetitionGroup;
+
+/**
+ *
+ */
+public class CompetitionGroupBoundary extends AbstractBoundary<
+        org.coner.api.entity.CompetitionGroup,
+        CompetitionGroup,
+        org.coner.hibernate.entity.CompetitionGroup> {
+
+    @Override
+    protected EntityMerger<org.coner.api.entity.CompetitionGroup, CompetitionGroup> buildApiToDomainMerger() {
```

Comment:
Maybe you need to submit a test, but I'm skeptical that the Api entity having a String for resultTimeType and the Domain entity having a ResultTimeType enum for resultTimeType.  I know the name of the getters/setters align, but will it do the conversion from String -> ResultTimeType enum -> String ?